GVFS/GVFS.FunctionalTests/Tools/RepositoryHelpers.cs

```diff
@@ -0,0 +1,20 @@
+﻿using GVFS.FunctionalTests.FileSystemRunners;
+using System.Runtime.InteropServices;
+
+namespace GVFS.FunctionalTests.Tools
+{
+    public static class RepositoryHelpers
+    {
+        public static void DeleteTestDirectory(string repoPath)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                CmdRunner.DeleteDirectoryWithUnlimitedRetries(repoPath);

```

Comment:
Add `// Use cmd.exe to delete the enlistment as it properly handles tombstones and reparse points` comment here as well